content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md

```diff
@@ -0,0 +1,240 @@
+
+
+* [Resource Cluster Profile Editor](/user-management/palette-rbac/project-scope-roles-permissions#clusterprofileeditor)
+
+
+* [Resource Cluster Profile Viewer](/user-management/palette-rbac/project-scope-roles-permissions#clusterprofileviewer)
+
+
+* [Resource Cluster Viewer](/user-management/palette-rbac/project-scope-roles-permissions#clusterviewer)
+
+###  Custom Permissions and Operations
+
+
+The set of customizable Permissions and Operations supported by Palette are as follows:
+
+<br />
+
+1. CloudAccount
+   * Cloudaccount.get
+   * cloudaccount.list
+
+
+2. CloudConfig  
+   * cloudconfig.delete  
+   * cloudconfig.get  
+   * cloudconfig.list  
+   * cloudconfig.update
+
+
+ 3. Cluster  
+    * cluster.delete  
+    * cluster.get  
+    * cluster.list  
+    * cluster.update
+   
+
+4. ClusterProfile  
+   * clusterProfile.delete  
+   * clusterProfile.get  
+   * clusterProfile.list  
+   * clusterProfile.publish  
+   * clusterProfile.update
+   
+
+5. DNS Mapping  
+   * dnsMapping.get  
+   * dnsMapping.list
+  
+
+6. Location  
+   * location.get  
+   * location.list
+
+
+7.  Machine  
+    * machine.get  
+    * machine.list
+
+
+8.  Macro  
+    * macro.get  
+    * macro.list
+
+
+9.  Registry  
+    * packRegistry.get  
+    * packRegistry.list
+
+
+
+
+To Assign resource role to the user:
+
+<br />
+
+* Login to Palette as Tenant Admin
+
+
+* Select the user to be assigned with role from the **Users & Teams** from the left main menu to go to **User Details**.
+
+
+* From the user details wizard, select **Resource Roles** Tab and click **+ New Resource Role**.
+
+
+* In the **Add Roles to User** wizard, enter the following details:
+  * **Projects**: The projects to which the user is assigned.
+  * **Filers**: Select the filters to be assigned from the drop-down. The Filters created will be displayed in the drop-down menu.
+  * Select the **check box** to assign the roles to the user from the list displayed. These are Palette built-in roles.
+
+
+* Click **Confirm** to complete the Add Role wizard.
+
+
+* The UI pop-up message will confirm the successful completion of the role assignment.
+
+## Assign Custom Resource Roles to the User
+
+Palette enables users to create `Custom Resource Roles` and assign them to the user. 
+
+<br />
+
+### Create Custom Roles
+
+To create your custom resource roles in Palette:
+
+* Log in to Palette as Tenant Admin and select **Roles** from the left main menu.
+
+
+* Click **+Create Resource Role** button to open the **Add New Role (Resource)** wizard. 
+  * Custom name for the role
+  * Select permissions and operations as per the user's requirements 
+
+
+* Once the required permissions are selected, click the **Save** button.
+
+
+* A UI pop-up message will confirm the successful creation of the new custom resource role.
+
+
+* Validate the role creation from **Roles** tab from the left main menu and go to **Resource Roles** tab. The New role will be listed on this page.
```

Comment:
```suggestion
* From the left main menu, click the **Resource Roles** tab to validate the new role is listed.
```

Reply:
It's unclear if the **Resource Roles** tab and **Roles** tab are the same thing. Offered suggestion above.